### DISCUSSION TITLE
Consumo giornaliero non si resetta

### DISCUSSION THREAD

**Discussion by GabboPi** [aturri/ha-zcsazzurro#25] [Q&A] [answered]
Buongiorno a tutti,

A qualcun altro risulta che le quantità giornaliere come "today imported power" non si resettino di giorno in  giorno, ma mostrino invece il totale da quando è stato avviato l'impianto? Se si, come avete risolto?
Stavo pensando di fare un piccolo script che calcolasse la differenza manualmente, di giorno in giorno, ma prima di cimentarmi volevo valutare altre soluzioni magari più furbe e veloci.

Grazie in anticipo a tutti!

**Comment by GabboPi** [comment]
Aggiungo uno screenshot per far capire meglio il problema: Inizialmente vengono caricati i consumi totali, poi durante la giornata solo i parziali (e chiaramente questi si vedono pochissimo perchè risultano trascurabili rispetto al totale). Idee su come aggiustare questo comportamento? Sto provando a usare degli utility meter ma non ho risolto granché 

![Screenshot 2023-10-21 alle 21 43 47](https://github.com/aturri/ha-zcsazzurro/assets/49364715/1b44c5e8-1259-4e34-b82f-0ff546139b7a)

**Reply by aturri** [reply, marked as answer]
Ciao,
Ti consiglio di scrivere al supporto ZCS. Quello che fa l'integrazione è interrogare il server ZCS terze parti e riportare i dati così come sono. Purtroppo il mio inverter non ha questo sensore, per cui non sono in grado di supportarti.

**Reply by GabboPi** [reply]
Grazie comunque: eventualmente li contatterò!
Oggi sembrava essersi sistemato da solo, nel caso avessi sviluppi utili li posterò qui.
